docs/docs/guides/troubleshooting.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Troubleshooting Neosync Jobs
+id: troubleshooting
+hide_title: false
+slug: /guides/troubleshooting
+---
+
+## Introduction
+
+Databases are _complex_. Neosync does its best to handle any and all edge cases you throw at it.
+
+This document is a collection of common issues you may face when running Neosync and how to resolve them.
+
+### `Sync: could not successfully complete sync activity: read tcp 0.0.0.0:42454->0.0.0.0:5432: i/o timeout`
+
+This is likely due to an overabundance of connections against the database. 
+
+You can resolve this by reducing the number of open connections in the "Connections" tab of Neosync. The default is at 80, the easiest way to debug is to start with something small like 10 and increase it until you hit the error again.
+
+### `could not successfully complete sync activity: pq: insert or update on table \"table_name\" violates foreign key constraint \"table_name_id_fkey\"`
+
+This is likely due to records in the source database, that are added _after_ the table on one side of the constraint is synchronized. If you have tables that are constantly in flux on the source database, and your job takes a long time to run this error happens more frequently.
+
+The easiest way to resolve it is to copy or snapshot your source connection to a fresh database with no other connections, and run the job again.
+
+### `ERROR: schema \"schema_name\" does not exist`
+
+Neosync does not support creating schemas on the fly. You must create the schema on the target database before running the job.
+
+
+### `unable to exec truncate cascade statements: ERROR: relation \"public.table_name\" does not exist`
```

Comment:
This is interesting because if you unselect truncate as well as don't init, but your table doesn't exist, you'll get a different error in that it can't insert because the table doesn't exist :)

Reply:
Yep - some of them are painfully obvious once you think about it